frontend/app/model/MembershipSummary.scala

```diff
@@ -0,0 +1,17 @@
+package model
+
+import org.joda.time.DateTime
+
+case class MembershipSummary(startDate: DateTime,
+                             firstPaymentEndDate: DateTime,
+                             amountPaidToday: Float,
+                             planAmount: Float,
+                             nextPaymentPrice: Float,
+                             nextPaymentDate: DateTime,
+                             renewalDate: DateTime,
+                             initialFreePeriodOffer: Boolean) {
+
+
+  val annual = startDate.plusYears(1) == renewalDate
```

Comment:
Okay, well let's deal with it as a separate PR as we have at least three places that does this `plusYear` or `plusDay` logic which is not great.